src/containers/appointments/Appointments.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ApptCard } from '../../components/apptCard/ApptCard';
+import { ApptPicker } from '../../components/apptPicker/ApptPicker';
+import { ApptsSubmitter } from '../../components/apptsSubmitter/ApptsSubmitter';
+
+export const Appointments = ({userType}) => {
+  let picker = <ApptPicker />
+  if (userType == 'Provider'){
+    picker = <ApptsSubmitter />
+  } 
+  
+  return (
+    <div>
+      <p>Appointments State: appointments </p>
+      <h1>{userType} Main Page</h1>
```

Comment:
This isn't semantically correct. An `h1` shouldn't have non-text children. Let's move `userType` outside of `h1`?

Reply:
I would like to know more about this. I thought it was fine to put text variables inside any text-based tags. Is it that it's a heading tag and this should be styled instead? Is the variable name poorly named?